aptos-move/framework/aptos-token/sources/token.move

```diff
@@ -847,6 +862,7 @@ module aptos_token::token {
         );
         token_data.supply = token_data.supply - burned_amount;
 
+        // TODO: What if the creator wants to mint more this token later?
```

Comment:
They have to recreate. We want to save the storage here as there. Otherwise, there would be a lot of non-used token data in creator's account

Reply:
Okay makes sense